loader/src/injector/jni_hooks.hpp

```diff
@@ -303,14 +377,19 @@ std::array nativeForkSystemServer_methods = {
         "(II[IIII[[IJJ)I",
         (void *) &nativeForkSystemServer_samsung_q
     },
+   JNINativeMethod {
+        "nativeForkSystemServer_grapheneos",
+        "(II[II[[IJJ)I",
+        (void *) &nativeForkSystemServer_grapheneos_u
+    }
 };
 
 } // namespace
 
 static void do_hook_zygote(JNIEnv *env) {
     vector<JNINativeMethod> hooks;
-    const char *clz;
-    clz = "com/android/internal/os/Zygote";
+    const char *clz = "com/android/internal/os/Zygote";
+
```

Comment:
`jni_hooks.cpp` was generated from `gen_jni_hooks.py` by executing`py gen_jni_hooks.py` for sure? asking so because it does not seem to be...

Reply:
Didn't look properly on the PR yet, but for the rest, yes

Reply:
i am doing this tests in another account of mine and then copy paste them here 
thanks for addressing this it's my mistake, sorry

Reply:
i fixed it